docs/api/io/BufferedSerial.md

```diff
@@ -0,0 +1,73 @@
+# BufferedSerial
+
+<span class="images">![](https://os.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_buffered_serial.png)<span>`BufferedSerial` class hierarchy</span></span>
+
+The `BufferedSerial` class provides UART functionality. We recommend you use this class for serial data transfers. You can use it to send and receive bytes of data in a sequence using separate transmit (TX) and receive pins (RX). Communication can be between two processors or for sending text to a console.
+
+Serial channels have the following characteristics:
+
+- TX and RX pins - can be specified as Not Connected (NC) for simplex (unidirectional) communication or as valid pins for full duplex (bidirectional) communication.
+- Baud rate - predefined speed at which data is sent and received. Standard baud rates include 9600, 119200 and 115200.
```

Comment:
Who or what sends the data? The BufferedSerial class?

Reply:
UART